### ISSUE TITLE
YouTube Restrict

### ISSUE THREAD

**Issue by test1964** [andybalholm/redwood#21]
Hey,

To enfoce  Google SafeSearch  I added in safesearch.conf file this line:

/google/d  safe=active

and it working very well.

How can I do safe search for Youtube that say to add this:

YouTube-Restrict: Strict

in the header(not in the end of the url)?

**Comment by andybalholm** [comment]
Header modifications are a feature that Redwood doesn't have yet.

**Comment by thinkwelltwd** [comment]
You can use [DNS](https://support.google.com/youtube/answer/6214622?hl=en)

**Comment by test1964** [comment]
Can you point where is code is the best way to handle it (want to try add this  for Youtube only)?

**Comment by andybalholm** [comment]
Did you read the page that thinkwelltwd linked to in his response?

**Comment by test1964** [comment]
Yes, but can't do it with DNS, I want to try to add it in code (with header way), Is there a simple way to put it code just for Youtube(even if it not nice for now)?

**Comment by thinkwelltwd** [comment]
> Is there a simple way to put it code just for Youtube(even if it not nice for now)?

That's not how the Redwood project has been developed. When there's a feature needed, and can't be met any other way, then the feature is considered. And @andybalholm then looks to produce a proper, elegant and performant way to add the feature, not merely bolt something on.

DNS is actually very simple and within the reach of even the smallest network to implement. It's not plausible that it can't be done.

**Comment by test1964** [comment]
Thanks
